neo4j-store/src/test/kotlin/org/ostelco/prime/storage/graph/Neo4jStoreTest.kt

```diff
@@ -964,15 +975,22 @@ class Neo4jStoreTest {
                                                             SimProfile(
                                                                     iccId = "iccId",
                                                                     eSimActivationCode = "eSimActivationCode",
-                                                                    status = AVAILABLE_FOR_DOWNLOAD))),
+                                                                    status = AVAILABLE_FOR_DOWNLOAD,
+                                                                    requestedOn = simProfileForNo.requestedOn
```

Comment:
Deep indentation.

Reply:
This deep indentation is as per code formatting settings of IDE.  This is not due to the deep nesting of control flow statements.

Reply:
Then the code settings are permitting code that is not very readable.  I will approve  in a few seconds, but this is not very readable code.